src/core/StickyContainer.tsx

```diff
@@ -23,6 +23,8 @@ export interface StickyContainerProps {
     stickyHeaderIndices?: number[];
     stickyFooterIndices?: number[];
     overrideRowRenderer?: (type: string | number | undefined, data: any, index: number, extendedState?: object) => JSX.Element | JSX.Element[] | null;
+    scrollOffsetCorrection?: () => number;
+    stickyContainerRenderer?: (stickyContent: JSX.Element, index: number, extendedState?: object) => JSX.Element | null;
```

Comment:
shouldn't there be a props section?

Reply:
Added the new params to propTypes, please refer to lines 238 - 243.